cyhy/core/geoloc.py

```diff
@@ -39,3 +40,12 @@ def lookup(self, ip):
             return (response.location.longitude, response.location.latitude)
         except AddressNotFoundError:
             return (None, None)
+
+    def check_restricted_ip(self, ip):
+        try:
+            response = self.__reader.city(str(ip))
+            if response.country.name in RESTRICTED_COUNTRIES:
+                return True,response.country.name
+            return False,None
+        except AddressNotFoundError:
+            print >> sys.stderr, "CIDR %s not found in geolocation database" % cidr
```

Comment:
So that this method has a return regardless of an exception. The return type for this method should be a `Typle[bool, str]`, so I replaced the `None` with an empty string while moving the return statement.

```suggestion
            print >> sys.stderr, "CIDR %s not found in geolocation database" % cidr

        return False, ""
```